src/SDKs/Search/DataPlane/Search.Tests/Tests/SkillsetsTests.cs

```diff
@@ -0,0 +1,610 @@
+            {
+                DefaultLanguageCode = keyPhraseLanguageCode,
+
+            });
+
+            return new Skillset("testskillset", "Skillset for testing", skills);
+        }
+
+        private static Skillset CreateTestSkillsetOcrSentiment(string ocrLanguageCode = "en", string sentimentLanguageCode = "en")
+        {
+            List<Skill> skills = new List<Skill>();
+
+            List<InputFieldMappingEntry> _inputs = new List<InputFieldMappingEntry>()
+            {
+                new InputFieldMappingEntry { Name = "url", Source = "/document/url" },
+                new InputFieldMappingEntry { Name = "queryString", Source = "/document/queryString" }
+            };
+
+            List<OutputFieldMappingEntry> _outputs = new List<OutputFieldMappingEntry>()
+            {
+                new OutputFieldMappingEntry { Name = "text", TargetName = "mytext" }
+            };
+
+            skills.Add(new OcrSkill("Tested OCR skill", PathSeparator + RootName, _inputs, _outputs)
+            {
+                TextExtractionAlgorithm = TextExtractionAlgorithm.Handwritten,
+                DefaultLanguageCode = ocrLanguageCode,
+
+            });
+
+            List<InputFieldMappingEntry> _inputs1 = new List<InputFieldMappingEntry>()
+            {
+                new InputFieldMappingEntry
+                {
+                    Name = "text",
+                    Source = "/document/mytext"
+                }
+            };
+
+            List<OutputFieldMappingEntry> _outputs1 = new List<OutputFieldMappingEntry>()
+            {
+                new OutputFieldMappingEntry
+                {
+                    Name = "score",
+                    TargetName = "mySentiment"
+                }
+            };
+
+            skills.Add(new SentimentSkill("Tested Sentiment skill", PathSeparator + RootName, _inputs1, _outputs1)
+            {
+                DefaultLanguageCode = sentimentLanguageCode,
+
+            });
+
+            return new Skillset("testskillset1", "Skillset for testing", skills);
+        }
+
+        private static Skillset CreateTestSkillsetImageAnalysisKeyPhrase()
+        {
+            List<Skill> skills = new List<Skill>();
+
+            List<InputFieldMappingEntry> _inputs = new List<InputFieldMappingEntry>()
+            {
+                new InputFieldMappingEntry { Name = "url", Source = "/document/url" },
+                new InputFieldMappingEntry { Name = "queryString", Source = "/document/queryString" }
+            };
+
+            List<OutputFieldMappingEntry> _outputs = new List<OutputFieldMappingEntry>()
+            {
+                new OutputFieldMappingEntry { Name = "description", TargetName = "mydescription" }
+            };
+
+            skills.Add(new ImageAnalysisSkill("Tested image analysis skill", PathSeparator + RootName, _inputs, _outputs)
+            {
+                VisualFeatures = new List<VisualFeatures?> { VisualFeatures.Description },
```

Comment:
You should include every `VisualFeatures` value to fully exercise serialization. Same for all enums.